helpers/guilds/voice/updateVoiceState.ts

```diff
@@ -10,7 +10,7 @@ import type { Bot } from "../../bot.ts";
  *  - You are able to set `request_to_speak_timestamp` to any present or future time.
  *  - When suppressed, the user will have their `request_to_speak_timestamp` removed.
  */
-export async function updateBotVoiceState(bot: Bot, guildId: bigint, options: UpdateSelfVoiceState): Promise<void> {
+export async function updateOwnVoiceState(bot: Bot, guildId: bigint, options: UpdateOwnVoiceState): Promise<void> {
```

Comment:
If this name is changed, id like to keep the `bot` name as an alias, i find that much easier to read and use than updateOwnxxx

Reply:
Added alias in latest commit.